nemo/collections/asr/modules/conv_asr.py

```diff
@@ -141,7 +141,7 @@ def __init__(
             residual_mode = lcfg.get('residual_mode', residual_mode)
             se = lcfg.get('se', False)
             se_reduction_ratio = lcfg.get('se_reduction_ratio', 8)
-            se_context_window = lcfg.get('se_context_window', -1)
+            se_context_window = lcfg.get('se_context_size', -1)
```

Comment:
All configs use se_context_size... but code checks se_context_window. Thankfully no major experiments were done with limited context. [BUGFIX]

Reply:
Well now that we have dataclasses, can we remove the .get()? I assume they now have smart defaults as opposed to not existing.

Reply:
No we cannot remove get. This is by design - 

1) We don't want to refactor old classes to support dataclasses. They are optional by design and dataclasses must interop with existing code, not the other way around. 

Since dataclasses are optional, users are still freely allowed to use yaml based configs, not just as overrides but as full configs. And therefore the get based methods are more stable c

Of course, if you use dataclasses, your yaml config gets validated during config merge.  So that's a useful benefit to having dataclasses. 

2) Updating all of our codebase to handle dataclasses as a special case required a massive effort. Instead, dataclasses are mapped down to dict configs immediately after they are passed to ModelPT so that they interop perfectly with existing code.